pycloudlib/azure/util.py

```diff
@@ -179,7 +179,7 @@ def is_pro_image(image_id, registered_image):
     if img_dict.get("publisher") == "Canonical":
         offer = img_dict["offer"]
     elif registered_image is not None:
-        offer = registered_image["offer"]
+        offer = registered_image["offer"] or ""
```

Comment:
Any reason we pefer this syntax to registered_image.get("offer", "") ?

Reply:
Yeah, I tried that syntax, but `offer` actually exists as a key, but the value is `None`. The return statement fails trying to iterate across None.